manifest/xsd/tract.xsd

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
These won't be present in the mobile app, or need to be used by the mobile app at all, this will be something for the API, and something to check our sample XML against as we are creating it